parts/k8s/cloud-init/artifacts/setup-custom-search-domains.sh

```diff
@@ -3,8 +3,24 @@ set -x
 source /opt/azure/containers/provision_source.sh
 
 echo "  dns-search <searchDomainName>" | tee -a /etc/network/interfaces.d/50-cloud-init.cfg
-systemctl_restart 20 5 10 restart networking
+systemctl_restart 20 5 10 networking
 wait_for_apt_locks
 retrycmd_if_failure 10 5 120 apt-get -y install realmd sssd sssd-tools samba-common samba samba-common python2.7 samba-libs packagekit
 wait_for_apt_locks
-echo "<searchDomainRealmPassword>" | realm join -U <searchDomainRealmUser>@$(echo "<searchDomainName>" | tr /a-z/ /A-Z/) $(echo "<searchDomainName>" | tr /a-z/ /A-Z/)
+
+function sourcekubeletscript() {
+  source /opt/azure/containers/kubelet.sh
+}
+
+function updatevariables() {
```

Comment:
I don't see this `updatevariables` func being called anywhere.

Reply:
Yes, this is only there so we can copy the function content to the new script realmjoin.sh because kublet.sh runs after setup-custom-search-domain.sh and calling it inside the script does not modify the content during the execution of it.